Gemfile

```diff
@@ -1,8 +1,11 @@
 source 'https://rubygems.org'
 gemspec
 
+gem 'actionview'
 gem 'activerecord'
 
+gem 'nokogiri', '~> 1.6.0' if RUBY_VERSION.start_with?('2.0')
```

Comment:
The latest version of nokogiri (1.7.x) requires Ruby 2.1 or higher, it's a test-only pin.